src/utils/graphql/mutations/updateReview.mutation.ts

```diff
@@ -0,0 +1,16 @@
+import { gql } from '@apollo/client'
```

Comment:
Slightly confused about the filename -- shouldn't it be `updateReviewResponses.mutation.ts`?

Reply:
Or should the query be called `updateReview` ?

Reply:
Doesn't seem like this query is being used anywhere. Is it in anticipation of later functionality, or has it been replaced by `updateReviewResponse`?

Reply:
Yea, it isn't very clear. I have create 2 mutations: `updateReview` which is updating a block of `review_responses` at the same time (used for the **Approve All** button) and another one `updateReviewResponse` to be used by a single decision updated - no in use yet.

I'll rename this one to: `updateReviewResponses` I hope it will be clearer.

Reply:
Deleted my previous reply - your are right that's not in use. I'll remove this mutation so we can create a proper one when we will update the review to be submitted.

Reply:
Done [d6ca19b](https://github.com/openmsupply/application-manager-web-app/pull/221/commits/d6ca19bfd79cb63d45aa620144da9f129ce427d2)